sample/ios-app/src/TestViewController.swift

```diff
@@ -25,11 +25,11 @@ class TestViewController: UIViewController {
         tableView.refreshControl = refreshControl
         
         
-        listViewModel.state.data().addObserver { data in
-            guard let units = data as? [TableUnitItem] else { return }
+        listViewModel.state.addObserver(observer: { state in
```

Comment:
you can use `data()` as it was - https://github.com/icerockdev/moko-mvvm/blob/master/mvvm-state/src/commonMain/kotlin/dev/icerock/moko/mvvm/livedata/StateLiveDataExt.kt#L9